### ISSUE TITLE
Fonts are blurred on 2k monitor, but there is no problem with 1080p monitor

### ISSUE THREAD

**Issue by ghost** [tauri-apps/tauri#919]
**Describe the bug**
Fonts are blurred on 2k monitor, but there is no problem with 1080p monitor。
The 1080p monitor is an external monitor for my laptop。
**To Reproduce**
```
clone the repo: git@github.com:TobiasTao/tauri-api-demos.git
yarn
yarn fe:dev
yarn be:dev
```

**Screenshots**
2k:
![image](https://user-images.githubusercontent.com/37654563/89121855-4d77e300-d4f5-11ea-9162-4e49e0ec8b91.png)
2k with chrome:
![image](https://user-images.githubusercontent.com/37654563/89122016-7187f400-d4f6-11ea-9da3-903cb9781a48.png)
1080p:
![image](https://user-images.githubusercontent.com/37654563/89121848-43ee7b00-d4f5-11ea-8510-ea3790a53148.png)

**Platform and Versions (please complete the following information):**
```
[tauri]: running info

Operating System - Windows_NT(10.0.18363) - win32/x64
Microsoft Edge - 44.18362.449.0_neutral__8wekyb3d8bbwe

Node.js environment
  Node.js - 12.18.3
  tauri.js - 0.11.0

Rust environment
  rustc - 1.45.0
  cargo - 1.45.0
  tauri-bundler - 0.9.1

Global packages
  NPM - 6.14.6
  yarn - 1.22.4

App directory structure
/.git
/.idea
/.vscode
/dist
/e2e
/node_modules
/src
/src-tauri

App
  tauri.rs - 0.9.0
  mode - embedded-server
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: http: https: 'unsafe-eval' 'unsafe-inline'
  distDir - ../dist/tauri-api-demos
  devPath - http://localhost:4200
```

**Comment by Miaourt** [comment]
I believe this is linked to https://github.com/webview/webview_rust/issues/49

**Comment by aquacash5** [comment]
Just adding my experience. 

Surface Pro 7

```
[tauri]: running info

Operating System - Windows_NT(10.0.18363) - win32/x64
Microsoft Edge - 44.18362.449.0_neutral__8wekyb3d8bbwe

Node.js environment
  Node.js - 14.12.0
  tauri.js - 0.12.0

Rust environment
  rustc - 1.46.0
  cargo - 1.46.0
  tauri-bundler - 0.9.3

Global packages
  NPM - 6.14.8
  yarn - 1.22.5

App directory structure
/dist
/elm-stuff
/node_modules
/src-elm
/src-tauri

App
  tauri.rs - 0.9.2
  mode - embedded-server
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: http: https: 'unsafe-eval' 'unsafe-inline'
  distDir - ../dist
  devPath - http://localhost:3000
```
![image](https://user-images.githubusercontent.com/10753165/94073023-ade91980-fdee-11ea-9585-298c3f4f2c26.png)

**Comment by victorhqc** [comment]
This can be fixed with a [workaround](https://github.com/tauri-apps/tauri/issues/1074#issuecomment-767539204) in any project using Tauri, might be a good idea to document this somewhere.

**Comment by nothingismagick** [comment]
I am closing this issue in favor of the one where @victorhqc filed a fix in #1074
